### ISSUE TITLE
CIFAR-10

### ISSUE THREAD

**Issue by qiaoviola** [gdwang08/STFPM#1]
您好，我想请问一下如果想用CIFAR-10数据集进行训练应该怎么更改网络的结构呢，期待您的回复

**Comment by gdwang08** [comment]
We simply follow the supervised setting to pre-train the teacher network using the cross-entropy loss. We use torchvision.models.ResNet18(pretrained=False, num_classes=10).

**Comment by qiaoviola** [comment]
非常感谢您的回复，但是我还是没有成功，请问可以看看您在MNIST和CIFAR-10运行的代码吗？（并且在测试中正常和异常数据的划分也遇到了一些问题）期待您的回复

**Comment by qiaoviola** [comment]
非常感谢您的回复，但是我还是没有成功，请问可以看看您在MNIST和CIFAR-10运行的代码吗？（并且在测试中正常和异常数据的划分也遇到了一些问题）期待您的回复
